# Dynamixel setup
motor_ids: [1, 2, 3, 4]
port: "/dev/ttyACM0"
baudrate: 1000000
control_mode: "position"

# Motion range limits 
# Motor Rotation Limits
min_position_rad:
  1: 3.1416    # 180 degrees in radians
  2: 1.5708
  3: 1.5708    # 90 degrees in radians
  4: 1.5708 
max_position_rad:
  1: 4.7124    # 270 degrees in radians
  2: 4.7124
  3: 3.1416    # 180 degrees in radians
  4: 4.7124

max_parallel_grip: 130

# Link lengths for IK or forward modeling
linkBase: 50
linkTact: 46
linkTip: 65

# Here we put the different gripper configurations
open_positions:
  1: 4.7124 
  2: 3.1416
  3: 1.5708 
  4: 3.1416